startup.sql

```diff
@@ -0,0 +1,41 @@
+/* This script creates the database for capurace. */
+/* The description of the database is in the wiki, Database-Structure.md */
+
+CREATE DATABASE IF NOT EXISTS capurace;
+USE capurace;
+
+CREATE TABLE IF NOT EXISTS user (
+        school VARCHAR(30) NOT NULL DEFAULT NULL UNIQUE KEY,
+        id SMALLINT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+        leader VARCHAR(10) NOT NULL DEFAULT NULL,
+        tel VARCHAR(11) NOT NULL DEFAULT NULL,
+        mail VARCHAR(30) NOT NULL DEFAULT NULL,
+        password VARCHAR(30) NOT NULL DEFAULT NULL,
+        nteams SMALLINT UNSIGNED NOT NULL DEFAULT 0,
+        bill INT NOT NULL DEFAULT 0,
+        paid SMALLINT NOT NULL DEFAULT 0
+);
```

Comment:
We also need a Boolean field `confirm` to indicate whether the account is confirmed by us, otherwise, he/she shouldn't be able to use most of the features. (A simple way to anti-spam).